editor/src/components/canvas/canvas-strategies/absolute-reparent-strategy.spec.tsx

```diff
@@ -174,8 +221,8 @@ describe('Absolute Reparent Strategy', () => {
                 position: 'absolute',
                 width: 20,
                 height: 30,
-                top: 15,
-                left: 40,
+                top: 30,
```

Comment:
were all these changes intentional?

Reply:
Yes. Until now this test did not really move the object, as it only tested reparenting itself, and that made the test cleaner. However, after this change the strategy does not reparent when drag is zero, so I needed to really drag, and update the coordinates according to that.